CHANGELOG.md

```diff
@@ -12,5 +12,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
In your commit message title a verb is missing and the scope is kind of duplicated, I would suggest:

> ✨(ashley) add custom user model, authentication backend and LTI handlers